src/CalculatorApi.h

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+
+#include <pistache/router.h>
+#include <pistache/endpoint.h>
+
+#include "CalculatorController.h"
+#include "HistoryController.h"
+
+using namespace Pistache;
+
+
+
+Rest::Router getRouter()
```

Comment:
If these are defined in .h files, they should be marked inline. Generally, though, you only want to inline simple functions/methods, like getters and setters. If they're more complicated, you should put them in a .cpp file. This function is pretty simple, so it's probably alright to inline it here. Making a separate .cpp file just for a couple functions can be a pain.